locust-metrics/locustfile.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+from uuid import uuid4
+import random
+from locust import HttpUser, task, between
+
+
+CAT_FRACTION = os.getenv("CAT_FRACTION", 0.75)
```

Comment:
How did cat fraction get this high? What is going on here?

Reply:
great point, let me change that, this is ridiculous: 🐶 > 🐱

Reply:
I think I deliberately snuck it in when I had a big PR